doc/release-notes-27217.md

```diff
@@ -0,0 +1,6 @@
+Wallet
+------
+
+* Purposes strings are now restricted to the currently known values of "send", "receive", and "refund".
+  Wallets that have unrecognized purpose strings will have loading warnings, and the `listlabels`
+  RPC will raise an error if an unrecognized purpose is requested.
```

Comment:
Perhaps mention that the “purpose string” appears in the context of addresses, at least for me that would not necessarily have been obvious just from this release note without reviewing the PR.

Reply:
Done.